ols/app/endpoints/ols.py

```diff
@@ -26,7 +26,19 @@
 auth_dependency = AuthDependency(virtual_path="/ols-access")
 
 
-@router.post("/query")
+query_responses = {
+    200: {
+        "description": "Query is valid and correct response from LLM is returned",
+        "model": LLMResponse,
+    },
+    500: {
+        "description": "Query can not be validated, LLM is not accessible or other internal error",
+        "model": ErrorResponse,
+    },
```

Comment:
any reason not to also list 401 + 403 errors as potential response types?

Reply:
we can, but I'm not sure about the model (i.e. about the actual format of generated error messages). Lemme check, I'll probably ask @nemerna